stroyprombeton/models.py

```diff
@@ -191,6 +186,102 @@ def save(self, *args, **kwargs):
         super().save(*args, **kwargs)
 
 
+class SectionQuerySet(models.QuerySet):
+
+    def bind_fields(self):
+        """Prefetch or select typical related fields to reduce sql queries count."""
+        return (
+            self.select_related('page')
+            .prefetch_related('products')
+        )
+
+    def active(self) -> 'SeriesQuerySet':
```

Comment:
We can reuse it from [there](https://github.com/fidals/refarm-site/blob/master/pages/models.py#L61)

Reply:
@artemiy312 , how we can do it? You referenced `PageQuerySet`. Current `SectionQuerySet` does not inherit it

Reply:
@duker33 , I was wrong